### PR TITLE
Added UI Library article

### PR DESCRIPTION
**Pull request by eshanrnh**

### CODE REVIEW

File: Extending/UI Library/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+With the UI Library, you get a collection of visual building blocks that consists of pieces to build any UI in Umbraco. Each component is a building block updating its display according to the data passed to it.
+
+## UI Library Storybook
+
+[Storybook](https://uui.umbraco.com/?path=/story/uui-avatar-group--aaa-overview&_ga=2.165515577.727907355.1637240820-415968107.1637240820) is an application that gathers all the components together, holds their documentation, and showcases different use case scenarios. You can explore all the components through stories reflecting their use cases.
+
+Each story has interactive controls that allows you to change the state of the component in real-time. Every publicly available property is editable in Storybook, so you can test out various configurations and use-cases.
+
+You can also change the stylesheet of custom properties, to see how the component will look like. Every story has a code example that you can copy and paste into your project, making it easy to implement the components in your own packages and extensions.
```
Comment (jonat123) [inline Extending/UI Library/index.md:17]:
```suggestion
# UI Library

The UI Library is a set of web components that can be used to build Umbraco User Interface. The UI Library separates the user interface from Umbraco’s business logic and creates a unified user experience, with coherent styling and naming, across all the Umbraco platforms and projects including the ones developed by you.

With the UI Library, you get a collection of visual building blocks that consists of pieces to build any UI in Umbraco. Each component is a building block updating its display according to the data passed to it.

## UI Library Storybook

[Storybook](https://uui.umbraco.com/?path=/story/uui-avatar-group--aaa-overview&_ga=2.165515577.727907355.1637240820-415968107.1637240820) is an application that gathers all the components together, holds their documentation, and showcases different use case scenarios. You can explore all the components through stories reflecting their use cases.

Each story has interactive controls that allow you to change the state of the component in real-time. Every publicly available property is editable in Storybook, so you can test out various configurations and use-cases.

You can also change the stylesheet of custom properties, to see how the component will look like. Every story has a code example that you can copy and paste into your project, making it easy to implement the components in your own packages and extensions.
```

**Comment by jonat123** [comment]
@eshanrnh Looks good, i have added a suggestion to the grammar :) 

Also is the UI library only available for V9 or is it also possible to use it for V8? 🤔

**Comment by eshanrnh** [comment]
> @eshanrnh Looks good, i have added a suggestion to the grammar :)
> 
> Also is the UI library only available for V9 or is it also possible to use it for V8? 🤔

@jonat123 The [post ](https://umbraco.com/blog/umbraco-ui-library-release/)doesn't mention v8 so I am not really sure.

**Comment by jonat123** [comment]
@eshanrnh  Looks good to me :D 

You are clear to merge it in 😁

**Pull request merged**